packages/postcss-svgo/src/index.js

```diff
@@ -48,14 +47,13 @@ function minify(decl, opts) {
       }
     }
 
-    if (!isSvg(svg)) {
-      return;
-    }
-
     let result;
     try {
       result = optimize(svg, opts);
       if (result.error) {
+        if (result.error.startsWith('Error in parsing SVG')) {
```

Comment:
I could not find a better way to detect the cause of the error. Maybe @TrySound has got a better suggestion? Anyway I wonder whether it makes sense to throw an error just because we cannot process sommething, instead of just leaving it alone. See this comment https://github.com/cssnano/cssnano/issues/910#issuecomment-811431880

Reply:
I think we don't need throw the error in this cases (and all cases, for example `postcss-calc`, if we can't parse it just log warning, so developers can report about it and we still continue work), let's use `console.warn` and continue work

Reply:
Doesn't postcss have `warn` method?

Reply:
oh, yep, forgot about it